File: mobile/src/main/java/info/papdt/express/helper/support/ClipboardUtils.java
package info.papdt.express.helper.support;

import android.content.ClipData;
import android.content.ClipboardManager;
import android.content.Context;

public class ClipboardUtils {

	public static void putString(Context context, String string) {
		ClipboardManager manager = (ClipboardManager) context.getSystemService(Context.CLIPBOARD_SERVICE);
		manager.setPrimaryClip(ClipData.newPlainText("text" , string));
	}

}

File: mobile/src/main/kotlin/info/papdt/express/helper/support/ClipboardUtils.kt
package info.papdt.express.helper.support

import android.content.ClipData
import android.content.ClipboardManager
import android.content.Context

object ClipboardUtils {

    fun putString(context: Context?, string: String?) {
        (context?.getSystemService(Context.CLIPBOARD_SERVICE) as? ClipboardManager)
                ?.primaryClip = string?.let { ClipData.newPlainText("text", it) }
    }

}
